src/main.zig

```diff
@@ -433,6 +433,7 @@ const usage_build_generic =
     \\  -framework [name]              (Darwin) link against framework
     \\  -F[dir]                        (Darwin) add search path for frameworks
     \\  -install_name=[value]          (Darwin) add dylib's install name
+    \\  --entitlements=[path]          (Darwin) add path to entitlements file for embedding in code signature
```

Comment:
Looks like it does not parse with an equal sign:

```suggestion
    \\  --entitlements [path]          (Darwin) add path to entitlements file for embedding in code signature
```